src/BaGet.Core/BaGet.Core.csproj

```diff
@@ -13,6 +13,7 @@
     <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="$(MicrosoftExtensionsPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.Options.ConfigurationExtensions" Version="$(MicrosoftExtensionsPackageVersion)" />
     <PackageReference Include="NuGet.Packaging" Version="$(NuGetPackageVersion)" />
+    <PackageReference Include="NuGet.Protocol" Version="$(NuGetPackageVersion)" />
```

Comment:
`NuGet.Protocol` depends on `NuGet.Packaging`, you should be able to delete line 15.

![image](https://user-images.githubusercontent.com/737941/109093629-85cb5280-76cd-11eb-9897-a567738dafcd.png)

Reply:
Done!